OpenRA.Game/Settings.cs

```diff
@@ -23,24 +23,52 @@ public enum MouseScrollType { Disabled, Standard, Inverted }
 
 	public class ServerSettings
 	{
+		[Desc("Sets the server name.")]
 		public string Name = "OpenRA Game";
+
+		[Desc("Sets the internal port.")]
 		public int ListenPort = 1234;
+
+		[Desc("Sets the advertised port.")]
 		public int ExternalPort = 1234;
+
+		[Desc("Reports the game to the master server list.")]
 		public bool AdvertiseOnline = true;
+
+		[Desc("Locks the game with a password.")]
 		public string Password = "";
+
 		public string MasterServer = "http://master.openra.net/";
-		public bool DiscoverNatDevices = false; // Allow users to disable NAT discovery if problems occur
-		public bool AllowPortForward = true; // let the user disable it even if compatible devices are found
+
+		[Desc("Allow users to disable NAT discovery if problems occur.")]
```

Comment:
This is incorrect - the setting *enables* the use of the NAT discovery.  It just happens to be disabled by default.